app.js

```diff
@@ -1,30 +1,133 @@
 import express from "express";
 import nunjucks from "nunjucks";
-import axios from "axios";
+import mongoose from "mongoose";
+import * as uuid from "uuid";
+import dotenv from "dotenv";
+import fs from "fs";
+import filterGameOptions from "./server/js/filterGameOptions.js";
+import wordCheck from "./server/js/wordCheck.js";
+import Game from "./server/models/Game.js";
+import HighScore from "./server/models/highScore.js";
+import saveInDb from "./server/js/saveInDB.js";
+import getOneFromDB from "./server/js/getOneFromDB.js";
+import updateOneFromDB from "./server/js/updateOneFromDB.js"
+import highScoreRouter from "./server/js/highScoreRouter.js";
+
+dotenv.config();
 
 const app = express();
 
-const env = nunjucks.configure('views', {
-    autoescape: true,
-    express: app
+const DBUrl = process.env.DB_URL;
```

Comment:
Great use of environment variables. I would recommend shutting down with instructions if this is missing, or add instructions to the README.